src/mui/field/ReferenceManyField.js

```diff
@@ -117,6 +117,9 @@ export class ReferenceManyField extends Component {
             basePath,
             isLoading,
         } = this.props;
+        if (!children) {
```

Comment:
SHould n't it be  if (!children || !React.Children.count(children)) { return }

Reply:
I don't want to throw an error in that case. But I want to keep it in case we have more than one children

Reply:
```js
 if (!children || !React.Children.count(children)) {
  return
}
if (React.Children.count(children) !== 1) {
  throw Error()
}
do_stuff()
```

Reply:
I don't see a case where I can encounter :  !React.Children.count(children)